redash/metrics/database.py

```diff
@@ -18,7 +18,7 @@ def _table_name_from_select_element(elt):
     if isinstance(t, Alias):
         t = t.original.froms[0]
 
-    while isinstance(t, _ORMJoin):
+    while isinstance(t, _ORMJoin) or isinstance(t, Join):
```

Comment:
Drive by fix as otherwise this would generate an error message in the logs when running the migration.